parse_requirements.sh

```diff
@@ -1,5 +1,5 @@
 
-[[ -f mirgecom/setup.py ]] || git clone https://github.com/illinois-ceesd/mirgecom
+[[ -f mirgecom/setup.py ]] || git clone -b $EMIRGE_MIRGECOM_BRANCH https://github.com/illinois-ceesd/mirgecom
```

Comment:
This will fail if EMIRGE_MIRGECOM_BRANCH isn't set when calling this script.

So I suggest to initialize the variable like this before this line:

```bash
EMIRGE_MIRGECOM_BRANCH ="${EMIRGE_MIRGECOM_BRANCH:-master}"
```